src/commands/compile/compile.ts

```diff
@@ -378,3 +381,15 @@ async function compileWeb(target: Target) {
       })
   }
 }
+
+const syncFolder = async (target: Target) => {
```

Comment:
In my opinion, this logic should belong to `copySyncFolder` function.

Reply:
added separate [comment](https://github.com/sasjs/cli/pull/1214#issuecomment-1158958162) on this PR